changelog/unreleased/pull-3935

```diff
@@ -0,0 +1,5 @@
+Enhancement: Only display the message if there were locks to be removed.
+
+`restic unlock` should only show `successfully removed locks` if there were locks to be removed.
+
+https://github.com/restic/restic/pull/3935
```

Comment:
The link to the issue is missing: `https://github.com/restic/restic/issues/3929`
And this changelog file should also be called `issue-3929`